kbase-extension/static/kbase/js/widgets/appWidgets2/view/sequenceView.js

```diff
@@ -56,20 +43,14 @@ define([
             model.value = newModel;
         }
 
-        function setModelValue(value, index) {
+        // TODO: actually make this update the set of inputs.
+        function setModelValue(value, display) {
```

Comment:
This never actually did anything. Also, to my knowledge, the `update` bus function never gets called on view widgets. So, lower priority for now.